dotnet/samples/Demos/TelemetryWithAppInsights/Program.cs

```diff
@@ -119,4 +227,39 @@ private static void LoadUserSecrets()
             .Build();
         TestConfiguration.Initialize(configRoot);
     }
+
+    private sealed class AIServiceSelector : IAIServiceSelector
```

Comment:
Why is this selector necessary?

Reply:
This is needed so that the kernel can pick the right services for the samples, e.g. AzureOpenAI, HuggingFace, and Gemini.